package.json

```diff
@@ -1,23 +1,36 @@
 {
   "name": "webpack",
   "version": "1.0.0",
-  "description": "",
+  "description": "<a name=\"readme-top\"></a>",
```

Comment:
- [ ] [OPTIONAL] I was wondering why you added this because it doesn't seem like the appropriate definition of your project. Doing things right helps to show your mastery and commitment to best practices. If you can't add a valid description of your project here, I think it's better to leave it blank. :wink: